### PR TITLE
Publish a notice that WICG/portals is inactive

### PR DESCRIPTION
**Pull request by jeremyroman**
This should make it clearer for anyone who comes by why there hasn't been recent activity, and points in some directions they could look to see where this work is continuing.

### CODE REVIEW

**Comment by jeremyroman** [comment]
@a4sriniv @domenic Make sense? Adithya suggested updating this repository after the related Chromium code was removed. Especially if there's anything else that should be included in this list.

**Comment by domenic** [comment]
Yeah, this makes sense to me. I wonder if we want to actually archive it?

**Comment by jeremyroman** [comment]
Maybe -- I don't know what that process entails or what its implications are. Do you happen to?

(it would be nice if it were still available and not a 404, for instance)

**Comment by lucasgadani** [comment]
LGTM

**Comment by domenic** [comment]
https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories . Note that you can always un-archive.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+The Portals proposal is not currently being actively pursued. 
+
+Many of the capabilities that Portals would have enabled are being explored in other proposals:
+* [navigation speculation](https://github.com/WICG/nav-speculation) (including speculation rules, prefetch, and prerendering)
```
Comment (a4sriniv) [inline README.md:4]: nit: should we link to https://wicg.github.io/nav-speculation/ for consistency with the fenced frame link below?

LGTM otherwise!

Comment (jeremyroman) [reply README.md:4]: That page is mostly an index, and I think the README.md gives a better overview of the work.

**Pull request merged**